neo/NeoSystem.cs

```diff
@@ -34,6 +28,13 @@ public NeoSystem(Store store)
         {
             this.store = store;
             Plugin.LoadPlugins(this);
+            ActorSystem = ActorSystem.Create(nameof(NeoSystem),
```

Comment:
And in this PR he changes it to the constructor.
Isn't this what we want?

Reply:
They are the same. Both of them run only once.